src/Services/Exporters/Rejected.php

```diff
@@ -30,7 +30,7 @@ public function __construct(DataImport $dataImport, User $user)
         $this->hashFilename = $this->hashFilename();
```

Comment:
wrap this in a job and run the job in batch->then(). This way the splitting job will not run the export rejected task, and the pipeline won't stay blocked.